packages/tools/viewer-alpha/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "@tools/viewer-alpha",
+    "version": "1.0.0",
+    "private": true,
+    "sideEffects": false,
+    "description": "A viewer using BabylonJS to display 3D elements natively",
```

Comment:
I guess it needs a readme.md for packaging. Is there any mandatory info we should put it?

Reply:
The readme.md file in the public viewer package would need to be updated - one for the current viewer and a new one for the new viewer. As part of the publishing process we will replace the files and the description on npmjs.com will be updated.

Reply:
Ok will circle back on this when we setup the public package.